inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/InAppMessagingSpec.kt

```diff
@@ -245,6 +244,13 @@ class InAppMessagingSpec : BaseTest() {
         LocalDisplayedMessageRepository.instance().numberOfTimesDisplayed(message) shouldBeEqualTo 0
     }
 
+    @Test
+    fun `should enable caching`() {
+        initializeInstance(true)
```

Comment:
`shouldDisableCaching` is set to true so why we expect the cache to be enabled?

Reply:
This should be fine since this is negated in `InAppMessaging.init()`.
But add new variable to avoid conflict.